sdk/textanalytics/Azure.AI.TextAnalytics/api/Azure.AI.TextAnalytics.netstandard2.0.cs

```diff
@@ -35,6 +35,7 @@ public partial class AnalyzeActionsResult
     {
         internal AnalyzeActionsResult() { }
         public System.Collections.Generic.IReadOnlyCollection<Azure.AI.TextAnalytics.AnalyzeSentimentActionResult> AnalyzeSentimentResults { get { throw null; } }
+        public System.Collections.Generic.IReadOnlyCollection<Azure.AI.TextAnalytics.ClassifyCustomCategoryActionResult> ClassifyCustomCategoryResults { get { throw null; } }
```

Comment:
The crew met with Johan yesterday and we agreed to rename to `ClassifyDocumentSingleCategory` instead. Do you plan to pick up the new names in this PR?

Reply:
Thanks Deya. I saw the renames, but I thought it might be wiser to wait until after today's archiboard meeting before applying them (Just in case any more changes in happen). Does that sound okay?